extensions/authy/src/component/OutputToCurrentApp.tsx

```diff
@@ -0,0 +1,25 @@
+import { ActionPanel, ActionPanelItemProps, closeMainWindow, environment, Icon, showHUD } from "@raycast/api";
+import { exec } from "child_process";
+import { ReactElement } from "react";
+
+type OutputToCurrentAppProps = { title: string, pin: string }
+
+export function OutputToCurrentApp({ title, pin }: OutputToCurrentAppProps): ReactElement<ActionPanelItemProps> {
```

Comment:
Why not use the `PasteAction`? https://developers.raycast.com/api-reference/user-interface/actions#pasteaction

Reply:
I can confirm `PastAction` works the same way

Reply:
I just want to output it, I don't want to mess with the pasteboard, I get I have the history, but often I have something in my current clipboard that I need and I'm using an OTP to login somewhere to paste it, it's not a big issue, but it is helpful to bypass that. But it's extra code that we don't really need so I will switch it over to use `PasteAction`